tests/sndrcv_tls_ossl_certless_serveronly.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+# alorbach, 2019-01-16
+# testing sending and receiving via OpenSSL TLS with anon auth using bare ipv4, no SNI
```

Comment:
I guess this comment is incorrect... - or what does "bare ipv4, no SNI" mean in this context?